.github/workflows/compatibility.yml

```diff
@@ -11,14 +11,14 @@ jobs:
       fail-fast:    false
       max-parallel: 10
       matrix:
-        woocommerce: [ '4.0.0', '4.3.0', '4.4.0', '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0' ]
+        woocommerce: [ '4.4.0', '4.5.0', '4.6.0', '4.7.0', '4.8.0', '4.9.0', '5.0.0', '5.1.0', '5.2.0', '5.3.0', '5.4.0', '5.5.0', '5.6.0', '5.7.0' ]
         wordpress:   [ 'latest' ]
         php:         [ '7.4' ]
         include:
           # Edge case: oldest dependencies compatibility
-          - woocommerce: '4.0.0'
+          - woocommerce: '4.4.0'
             wordpress:   '5.4'
-            php:         '7.0'
+            php:         '7.1'
```

Comment:
Since we no longer run PHPUnit with PHP 7.0, I think this line below should be removed too: 

```bash
- run: if [[ "${{ matrix.php }}" == '7.0' ]]; then wget https://phar.phpunit.de/phpunit-6.5.14.phar && mv phpunit-6.5.14.phar phpunit.phar; fi;
```

Reply:
+1 here

Reply:
I think it's ok for now, but we'd need to put it back once/if we solve plugin loading issue for PHP 7 & PHP Unit 6.5.14

Reply:
Commented out this line, and added todos for this line. Ref: 0f234a3